agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -149,8 +168,9 @@ static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_
     son_slave_conf.hostap_iface = beerocks_slave_conf.hostap_iface[slave_num];
     son_slave_conf.hostap_ant_gain =
         beerocks::string_utils::stoi(beerocks_slave_conf.hostap_ant_gain[slave_num]);
-    son_slave_conf.radio_identifier        = beerocks_slave_conf.radio_identifier[slave_num];
-    son_slave_conf.backhaul_wireless_iface = beerocks_slave_conf.sta_iface[slave_num];
+    son_slave_conf.radio_identifier = beerocks_slave_conf.radio_identifier[slave_num];
+    son_slave_conf.backhaul_wireless_iface =
+        get_sta_iface_from_hostap_iface(son_slave_conf.hostap_iface);
```

Comment:
Hm, I'm not 100% sure if this is OK. bpl supposedly should only be used from the platform_manager thread (which BTW is going to become a separate process - cfr. #575). It is still OK-ish to do it from the main process, but I'm a bit concerned about adding calls to the son_slave - currently there aren't any.

So I think this should instead be part of the wlan_settings in platform_settings and sent by the platform manager in fill_platform_settings.

Clearly this is something that we can handle later.